.github/workflows/automerge.yml

```diff
@@ -0,0 +1,10 @@
+name: automerge
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '42 2,5,8,11 * * *'
```

Comment:
Any specific reason you chose 2:42, 5:42, 8:42, and 11:42 to run this cron job?  Also, why is this run only in the morning and not the afternoon?

Reply:
> Any specific reason you chose 2:42, 5:42, 8:42, and 11:42 to run this cron job? Also, why is this run only in the morning and not the afternoon?

Copy/paste from dx-core and the number 42 comes from [The Hitch Hiker's Guide to the Galaxy](https://www.scientificamerican.com/article/for-math-fans-a-hitchhikers-guide-to-the-number-42)

Reply:
> It looks like Dependabot is only allowed to create PRs on Saturday. In that case, does the automerge need to run every day, or can it run only on Saturday?

Automerge is run daily within the times noted. First, to take pressure off of CI/CD. After 12 is when developers are awake and being productive ;) 

It is run daily to allow for dependabot PRs to be fixed, if checks don't pass, which is also dependent on developers being awake and being productive ;)

To do a positive spin, create a reasonable amount of dependabot PRs on Saturday and give devs a week to get them merged with clean checks. The hope is there will not need to be a whole lot of developer intervention, because by Monday all of the dependabot PRs are green and merged.